addons/l10n_es/data/account_type.xml

```diff
@@ -4,13 +4,11 @@
     <!-- Actualizar método de cierre de tipos de cuenta estándar -->
     
     <data noupdate="1">
-        <record id="account.data_account_type_income"
-            model="account.account.type">
+        <record id="account.data_account_type_income" model="account.account.type">
             <field name="name">Income</field>
```

Comment:
your patch does not modify the existing behavior of these lines but I really wonder why you need them: don't you reset your P&L at the beginning of a new fiscal year? 

If yes, I don't get how you're dealing with the closing/opening entries given the close method of income/expense accounts set to balance.